### PR TITLE
Pull request for space travelers hub project

### PR DESCRIPTION
**Pull request by Harrynoqs**
In this project we incorporated the following:

- Working with the real live data from the SpaceX API is done.
- A web application for a company that provides commercial and scientific space travel services is built.
- The application will allow users to book rockets and join selected space missions.
- The Space Travelers' Hub consists of Rockets, Missions, and the My Profile section.
- The Rockets section displays a list of all available SpaceX rockets. Users can book each rocket by clicking the reservation 
   button or cancel the previously made booking.
- The Missions section displays a list of current missions along with their brief description and participation status. There is also 
   a button next to each mission that allows users to join the selected mission or leave the mission the user joined earlier.
- The My Profile section displays all reserved rockets and space missions.

### CODE REVIEW

File: src/tests/Rockets.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { Provider } from 'react-redux';
+import Rockets from '../components/Rockets';
+import store from '../redux/store';
+
+describe('Rockets tests', () => {
+  test('should render', () => {
+    const rockets = render(
+      <Provider store={store}>
+        <BrowserRouter>
+          <Rockets />
```
Comment (frank1738) [inline src/tests/Rockets.test.js:13]: - [ ] Your tests are not passing. Kindly try to fix this error for your tests to pass.

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi Team, 

Wonderful job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ App is designed as expected 👍 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tests/Rockets.test.js (hunk 1 above)
Comment (chuaindia) [reply src/tests/Rockets.test.js:13]: All the tests are passed now.

**Review by EricMbouwe — APPROVED** [review]
Hi @Harrynoqs, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Requested changes ✔
- No linter errors ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**